netjsonconfig/schema.py

```diff
@@ -443,11 +451,83 @@
                 }
             }
         },
+        "encryption_mfp_property": {
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["0", "1", "2"],
+                    "options": {"enum_titles": ["disabled", "optional", "required"]},
+                    "propertyOrder": 4,
+                }
+            }
+        },
+        "encryption_mfp_property_required": {
+            "required": ["ieee80211w"],
+            "properties": {
+                "ieee80211w": {
+                    "type": "string",
+                    "title": "management frame protection",
+                    "enum": ["2"],
```

Comment:
We can also add `"readOnly": True,` to make this option disabled in Configuration Editor of openwisp-controller. 
Same applies for `encryption_mfp_property_optional`. 

![Screenshot from 2022-02-21 23-36-53](https://user-images.githubusercontent.com/32094433/155007350-bd996676-8b72-43b5-b693-07748da503d8.png)